docs/01_nodeos/02_usage/03_development-environment/01_local-multi-node-testnet.md

```diff
@@ -90,7 +90,9 @@ This creates a special producer, known as the "bios" producer. Assuming everythi
 
 ### 5. Start the Second Producer Node
 
-The following commands assume that you are running this tutorial from the `eos\build` directory, from which you ran `./eosio_build.sh` to build the EOSIO binaries.
+<!-- don't render for now
```

Comment:
This will break the MDX/ doc6s parsing. Need to use this format `[//]: # ( THIS IS A COMMENT FOLLOWING LINE HAD JS COMMENTS JS COMMENTS BREAK MDX ) `.